src/App.js

```diff
@@ -12,7 +12,7 @@ function App() {
 	return (
 		<Router>
 			<div className="App">
-				<img src={logo} className="logo" />
+				<img src={logo} className="logo" alt="logo" />
```

Comment:
I know Is not you that added the logo there, but can you please create a separate component for the logo. In case we want to reuse the logo in different places.

Reply:
sure no problem